test/object-store/sync/sync_test_utils.cpp

```diff
@@ -304,10 +304,10 @@ struct FakeLocalClientReset : public TestClientReset {
             }
             remote_realm->commit_transaction();
 
-            TestLogger logger;
             sync::SaltedFileIdent fake_ident{1, 123456789};
             auto local_db = TestHelper::get_db(local_realm);
             auto remote_db = TestHelper::get_db(remote_realm);
+            util::NullLogger logger;
```

Comment:
This prevents anything from being logged in `perform_client_reset_diff`, potentially losing important debug information.

Reply:
Updated to a `StderrLogger` so we get the output